### PR TITLE
fix: (Icon) search & table icon update

### PR DESCRIPTION
**Pull request by jaanus03**
This PR updates search & table icons:

**Old search icons**
screenshot 2019-03-04 at 17 28 15

**New search icons**
screenshot 2019-03-04 at 17 22 10

**Old table icons**
screenshot 2019-03-04 at 17 26 53

**New table icons**
screenshot 2019-03-04 at 17 24 29

### CODE REVIEW

File: packages/react/src/themes/teams/components/Icon/svg/icons/search.tsx [hunk 1]
```diff
@@ -1,10 +1,21 @@
 import * as React from 'react'
+import cx from 'classnames'
 import { TeamsProcessedSvgIconSpec } from '../types'
+import { teamsIconClassNames } from '../teamsIconClassNames'
 
 export default {
   icon: ({ classes }) => (
     <svg role="presentation" focusable="false" viewBox="8 8 16 16" className={classes.svg}>
-      <path d="M22.9 22.2l-4.4-4.4c.8-.9 1.3-2.1 1.3-3.4 0-3-2.4-5.4-5.4-5.4-3 0-5.4 2.4-5.4 5.4 0 3 2.4 5.4 5.4 5.4 1.3 0 2.5-.5 3.4-1.3l4.4 4.4c.1.1.2.1.3.1.1 0 .2 0 .3-.1.2-.2.2-.5.1-.7zm-13-7.8c0-2.4 2-4.4 4.4-4.4s4.4 2 4.4 4.4-2 4.4-4.4 4.4-4.4-2-4.4-4.4z" />
+      <g>
```
Comment (codepretty) [inline packages/react/src/themes/teams/components/Icon/svg/icons/search.tsx]: `g` isn't necessary here

File: packages/react/src/themes/teams/components/Icon/svg/icons/table.tsx [hunk 2]
```diff
@@ -5,18 +5,16 @@ import { teamsIconClassNames } from '../teamsIconClassNames'
...
   icon: ({ classes }) => (
-    <svg
-      role="presentation"
-      focusable="false"
-      viewBox="8 8 16 16"
-      className={classes.svg}
-      aria-labelledby="icons_insert_table"
-    >
+    <svg role="presentation" focusable="false" viewBox="8 8 16 16" className={classes.svg}>
       <g>
```
Comment (codepretty) [inline packages/react/src/themes/teams/components/Icon/svg/icons/table.tsx]: `g` isn't necessary here

**Review by codepretty — APPROVED** [review]
Looks good to me - just remember to add a changelog entry

File: packages/react/src/themes/teams/components/Icon/svg/icons/search.tsx (hunk 1 above)
Comment (jaanus03) [reply packages/react/src/themes/teams/components/Icon/svg/icons/search.tsx]: Good catch. Removed them.

**Pull request merged**